src/app_service/service/network/dto.nim

```diff
@@ -15,9 +15,11 @@ type NetworkDto* = ref object
   nativeCurrencySymbol* {.serializedFieldName("nativeCurrencySymbol").}: string
   isTest* {.serializedFieldName("isTest").}: bool
   enabled* {.serializedFieldName("enabled").}: bool
+  chainColor* {.serializedFieldName("chainColor").}: string
+  shortName* {.serializedFieldName("shortName").}: string
 
 proc `$`*(self: NetworkDto): string =
-  return fmt"Network(chainId:{self.chainId}, name:{self.chainName}, rpcURL:{self.rpcURL}, isTest:{self.isTest}, enabled:{self.enabled})"
+  return fmt"Network(chainId:{self.chainId}, nativeCurrencyDecimals:{self.nativeCurrencyDecimals}, layer:{self.layer}, chainName:{self.chainName}, name:{self.chainName}, rpcURL:{self.rpcURL}, blockExplorerURL:{self.blockExplorerURL}, iconURL:{self.iconURL}, nativeCurrencyName:{self.nativeCurrencyName}, nativeCurrencySymbol:{self.nativeCurrencySymbol}, isTest:{self.isTest}, enabled:{self.enabled}, chainColor:{self.chainColor}, shortName:{self.shortName})"
```

Comment:
@Khushboo-dev-cpp just for the sake of readability if you want you can format it like this:
```
return fmt"""Network(
  chainId:{self.chainId}, 
  nativeCurrencyDecimals:{self.nativeCurrencyDecimals}, 
  layer:{self.layer}, 
  chainName:{self.chainName}, 
  name:{self.chainName}, 
  rpcURL:{self.rpcURL}, 
  blockExplorerURL:{self.blockExplorerURL}, 
  iconURL:{self.iconURL}, 
  nativeCurrencyName:{self.nativeCurrencyName}, 
  nativeCurrencySymbol:{self.nativeCurrencySymbol}, 
  isTest:{self.isTest}, enabled:{self.enabled}, 
  chainColor:{self.chainColor}, 
  shortName:{self.shortName}
)"""
```

Reply:
done